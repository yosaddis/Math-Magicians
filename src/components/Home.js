import React from 'react';

export default function Home() {
  return (
    <div style={{ width: '54rem', margin: '0 auto' }}>
      <h1 style={{ color: '#f5913e' }}>Welcome to our page</h1>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Enim illum vel
        modi cupiditate, voluptatem vitae consequuntur, culpa quidem eligendi
        pariatur nostrum minima labore ipsum. Error labore ex illo libero esse.
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis animi
        ut asperiores consequuntur? Molestiae obcaecati esse nostrum molestias
        non officia dolorum magnam natus saepe recusandae blanditiis, doloremque
        dolores! Veniam, a.
      </p>
    </div>
  );
}
