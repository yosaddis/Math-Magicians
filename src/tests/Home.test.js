import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import Home from '../components/Home';

describe('Unit test for home component', () => {
  test('The title in Home component should be rendered correctly.', () => {
    render(<Home />);
    expect(screen.getByText('Welcome to our page')).toBeInTheDocument();
  });
  test('Home component should be rendered correctly.', () => {
    const home = render(<Home />);
    expect(home).toMatchSnapshot();
  });
});
