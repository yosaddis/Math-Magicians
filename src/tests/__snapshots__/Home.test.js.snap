// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit test for home component Home component should be rendered correctly. 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        style="width: 54rem; margin: 0px auto;"
      >
        <h1
          style="color: rgb(245, 145, 62);"
        >
          Welcome to our page
        </h1>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Enim illum vel modi cupiditate, voluptatem vitae consequuntur, culpa quidem eligendi pariatur nostrum minima labore ipsum. Error labore ex illo libero esse. Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis animi ut asperiores consequuntur? Molestiae obcaecati esse nostrum molestias non officia dolorum magnam natus saepe recusandae blanditiis, doloremque dolores! Veniam, a.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      style="width: 54rem; margin: 0px auto;"
    >
      <h1
        style="color: rgb(245, 145, 62);"
      >
        Welcome to our page
      </h1>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Enim illum vel modi cupiditate, voluptatem vitae consequuntur, culpa quidem eligendi pariatur nostrum minima labore ipsum. Error labore ex illo libero esse. Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis animi ut asperiores consequuntur? Molestiae obcaecati esse nostrum molestias non officia dolorum magnam natus saepe recusandae blanditiis, doloremque dolores! Veniam, a.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
